_pages/avenger96-iot-box-giveaway.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Avenger96/IoT Box Giveaway
+permalink: /avenger96-iot-box-giveaway/
+js-package: giveaway
+css-package: giveaway
+description: >
+    96Boards have teamed up with Arrow, ST and Shiratech to give you the chance to get a free Avenger96 or IoT Box. This is an online campaign dedicated to Arrow's 96Boards products, main and mezzanine, around STM's MP1 96Boards.
+layout: flow
+jumbotron:
+    title: Avenger96 / IoT Box Giveaway
+    description: >
+        96Boards have teamed up with Arrow, ST and Shiratech to give you the chance to get a free Avenger96 or IoT Box. This is an online campaign dedicated to Arrow's 96Boards products, main and mezzanine, around STM's MP1 96Boards.
+flow:
+    - row: container_row
+      sections:
+        - format: members
+          style: zoom
+          members_content:
+            item_width: 3
+            items:
+              - name: Arrow
+                image:
```

Comment:
Are these images supposed to link to websites? They react to mouse hovering over in the deploy view